pkg/agent/config_test.go

```diff
@@ -103,36 +99,12 @@ func TestInvalidConfigError(t *testing.T) {
 	}
 }
 
-func TestMissingConfigError(t *testing.T) {
```

Comment:
This test is still a valid test after deleting the token, isn't it?

Reply:
Yeah, for some reason I thought that org and cluster ids conditionals weren't needed. But the other conditions in the test don't really throw any error if they are missing

Reply:
I'll add it back in the next commit since it's valid